Added details on port configuration (#926)
PegaSysEng/pantheon: docs/Reference/Pantheon-CLI-Syntax.md

metrics-port="6174"
```

> Specifies the port(TCP) on which [Prometheus](https://prometheus.io/) accesses [Pantheon metrics](../Using-Pantheon/Debugging.md#monitor-node-performance-using-prometheus).

Reviewer: ```suggestion
Specifies the port (TCP) on which [Prometheus](https://prometheus.io/) accesses [Pantheon metrics](../Using-Pantheon/Debugging.md#monitor-node-performance-using-prometheus).
```

Author: Updated

metrics-push-port="6174"
```

> Port(TCP) of the [Prometheus Push Gateway](https://github.com/prometheus/pushgateway).

Reviewer: ```suggestion
Port (TCP) of the [Prometheus Push Gateway](https://github.com/prometheus/pushgateway).
```

Author: Updated

p2p-port="1789"
```

> Specifies the P2P peer discovery listening port (UDP and TCP).

Reviewer: ```suggestion
Specifies the P2P listening ports (UDP and TCP).
```

Reviewer: To clarify - this doesn't just set the discovery listening port, so I think its more accurate to leave out "peer discovery".

Author: Updated

```

> Specifies the port on which P2P peer discovery listens.

Reviewer: This port is used for both discovery and wire protocol communication.  Maybe change this to something like: "Specifies the UDP port on which the devP2P discovery protocol listens and the TCP port on which the devP2P wire protocol listens."  Or something simpler like "Specifies p2p listening ports (UDP and TCP)".

Author: Updated

> The default is 30303. The peer discovery port must be open on both TCP and UDP.

Reviewer: I don't think I'd add these comments here - it assumes there's a firewall set up so its not really a descriptive comment about the config but assumes a certain networking environment.

Author: Added link to port configuration section
